package.json

```diff
@@ -188,13 +192,29 @@
                 "command": "language-julia.executeJuliaCellInREPL",
                 "title": "Julia: Execute Code Cell"
             },
+            {
+                "command": "language-julia.executeJuliaCellInREPLAndMove",
+                "title": "Julia: Execute Code Cell And Move"
+            },
             {
                 "command": "language-julia.executeJuliaBlockInREPL",
                 "title": "Julia: Execute Code Block"
             },
             {
                 "command": "language-julia.selectBlock",
                 "title": "Julia: Select Code Block"
+            },
+            {
+                "command": "language-julia.removeAllInlineResults",
+                "title": "Julia: Remove All Inline Results"
+            },
+            {
+                "command": "language-julia.removeAllInlineResultsInEditor",
+                "title": "Julia: Remove Inline Results In Editor"
+            },
+            {
+                "command": "language-julia.removeCurrentInlineResult",
+                "title": "Julia: Remove Current Inline Result"
```

Comment:
nitpick, how about renaming these commands so that we use "clear" instead of "remove" ?
ink exports these functionalities under the convention so it may sound more natural for Juno users.